infra/terraform/templates/alb.tf

```diff
@@ -30,3 +39,16 @@ resource "aws_alb_listener" "wellcomecollection" {
     type             = "forward"
   }
 }
+
+resource "aws_alb_listener" "wellcomecollection_https" {
+  load_balancer_arn = "${aws_alb.wellcomecollection_alb.id}"
+  port              = 443
+  protocol          = "HTTPS"
+  ssl_policy        = "ELBSecurityPolicy-2015-05"
```

Comment:
Just out of interest why ELBSecurityPolicy-2015-05? and not ELBSecurityPolicy-2016-08?